data/event-schemas/schema/m.room.power_levels.yaml

```
@@ -42,6 +42,9 @@ description: |-
   `m.room.power_levels` event as soon as it is created. See also the
   documentation of the `/createRoom` API.
 
+  Note: the allowed range for power level values is `[-(2**53)+1, (2**53)-1]`,
+  as specified by the [Canonical JSON specification](/appendices/#canonical-json).
```

lots of "specification"

```suggestion
  as required by the [Canonical JSON specification](/appendices/#canonical-json).
```